WebExample/tests/styles.spec.ts

```diff
@@ -0,0 +1,70 @@
+import {test, expect} from '@playwright/test';
+import type {Page} from '@playwright/test';
+import * as CONSTANTS from '../../constants';
+
+const setupInput = async (page: Page) => {
```

Comment:
I saw a similar function in other test files, can we move it to the utils file and reuse it in every component?